hack/wait-for-kudo.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
I doubt that this script will work with `sh`. And on most systems `/bin/sh` is running `/bin/bash` anyways.
```suggestion
#!/usr/bin/env bash
```